Src/Plugins/Security/SEC102.ReviewPotentiallySensitiveData.json

```diff
@@ -22,6 +24,13 @@
           "Name": "ReviewPotentiallySensitiveData/SocialSecurityNumber",
           "ContentsRegex": "\\b(?P<refine>(?P<secret>[0-9]{3}-[0-9]{2}-[0-9]{4}))\\b",
           "MessageArguments": { "dataKind": "social security number" }
+        },
+        {
+          "Id": "SEC102/003",
+          "Name": "ReviewPotentiallySensitiveData/Url",
+          "FileNameAllowRegex": "(?i)(\\.cs|\\.cpp|\\.ini|\\.js|\\.ps1)$",
```

Comment:
>FileNameAllowRegex [](http://example.com/codeflow?start=11&length=18)

This extensions list is a start to use when we run our investigative analysis. It's not clear this is what we'll want moving forward. Can discuss with you offline.

Reply:
@michaelcfanning Is this comment with respect to the URL analysis or the certificate analysis?

Reply:
URL